opennms-dao/src/test/java/org/opennms/netmgt/dao/support/DefaultResourceDaoTest.java

```diff
@@ -268,6 +269,10 @@ public void testGetTopLevelResourceDomainDoesNotExistInCollectdConfig() {
         assertNull("Resource should be null", resource);
     }
 
+    /**
+     * Rewrite of RmotePoller is still in progress, so ignore this one for now.
+     */
+    @Ignore
```

Comment:
Please add an `TODO cpape` so we can find this later on.

Reply:
done